README.md

```diff
@@ -2,14 +2,21 @@
 # io-pay
 - [io-pay](#io-pay)
   - [Prerequisites](#prerequisites)
+  - [Documentation](#documentation)
   - [Running](#running)
+  - [Sequence diagram : _WISP light_](#sequence-diagram--wisp-light)
   
 Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
 
 ## Prerequisites
 
 - [yarn](https://classic.yarnpkg.com/en/docs/getting-started)
 
+## Documentation
+All documentary resources can be found here 🚀
+- [WISP light](https://drive.google.com/drive/u/0/folders/19LzYVs__dwzCYPtUli_7FpP4wbBQMIVh)
+- [io-pay UX MVP](https://pagopa.invisionapp.com/console/share/4A1RGFXIQM)
+
```

Comment:
@GiovanniMancini in order to have all documentation reference handy, could be useful refer/link here the documentation you are reviewing too ?